src/parameterizations/stochastic/MOM_stochastics.F90

```diff
@@ -138,10 +256,202 @@ subroutine update_stochastics(CS)
   call callTree_enter("update_stochastics(), MOM_stochastics.F90")
 
 ! update stochastic physics patterns before running next time-step
-  call run_stochastic_physics_ocn(CS%sppt_wts,CS%epbl1_wts,CS%epbl2_wts)
+  call run_stochastic_physics_ocn(CS%sppt_wts,CS%skeb_wts,CS%epbl1_wts,CS%epbl2_wts)
+
+  call callTree_leave("update_stochastics(), MOM_stochastics.F90")
 
-  return
 end subroutine update_stochastics
 
+subroutine apply_skeb(grid,GV,CS,uc,vc,thickness,tv,dt,Time_end)
+
+  type(ocean_grid_type),   intent(in)    :: grid   !< ocean grid structure
+  type(verticalGrid_type), intent(in)    :: GV     !< ocean vertical grid
+  type(stochastic_CS),     intent(inout) :: CS     !< stochastic control structure
+
+  real, dimension(SZIB_(grid),SZJ_(grid),SZK_(GV)), intent(inout) :: uc        !< zonal velocity [L T-1 ~> m s-1]
+  real, dimension(SZI_(grid),SZJB_(grid),SZK_(GV)), intent(inout) :: vc        !< meridional velocity [L T-1 ~> m s-1]
+  real, dimension(SZI_(grid),SZJ_(grid),SZK_(GV)),  intent(in)    :: thickness !< thickness [H ~> m or kg m-2]
+  type(thermo_var_ptrs),                            intent(in)    :: tv       !< points to thermodynamic fields
+  real,                                       intent(in)    :: dt       !< time increment [T ~> s]
+  type(time_type),                            intent(in)    :: Time_end !< Time at the end of the interval
+! locals
+
+  real ALLOCABLE_, dimension(NIMEMB_PTR_,NJMEMB_PTR_,NKMEM_) :: psi
```

Comment:
I don't understand why `psi`, `ustar`, `vstar` and `diss_tmp` are allocatable arrays here.  If this routine is called, these arrays are used, so why not just declare them the same way as local variables are everywhere else in the code, e.g., `real :: psi(SZIB_(grid),SZJB_(grid),SZK_(GV)) ! A descriptive comment with units`.

  Also, there need to be comments added describing  `psi`, `ustar`, `vstar` and `diss_tmp` and their units, as requested at https://github.com/mom-ocean/MOM6/wiki/Code-style-guide#documentation-in-code.